ebos/eclproblem.hh

```diff
@@ -3072,6 +3085,8 @@ private:
     struct PffDofData_
     {
         Opm::ConditionalStorage<enableEnergy, Scalar> thermalHalfTrans;
+        Opm::ConditionalStorage<enableDiffusion, Scalar> diffusivity;
+        //Scalar diffusivity;
```

Comment:
Is this comment supposed to be a hint/explanatory?  If not, maybe we can drop it altogether?

Reply:
Just a leftover. I will remove it.